cmake/org.eclipse.cdt.cmake.core/src/org/eclipse/cdt/cmake/core/CMakeBuildConfiguration.java

```diff
@@ -162,7 +203,8 @@ public IProject[] build(int kind, Map<String, String> args, IConsole console, IP
 				runCMake = true;
 			}
 
-			ICMakeProperties cmakeProperties = getPropertiesController().load();
+			ICMakeProperties cmakeProperties = getCMakeProperties();
```

Comment:
@jonahgraham ,
Now that getPropertiesController() is no longer called, the Runnable cmakeCacheDirtyMarker param to CMakePropertiesController is not setup and thereby this code is not executed:

deleteCMakeCache = true;

I'm not sure if this matters.

Reply:
All that dead code should be deleted along with my deletion comment on `ICMakePropertiesController` as it all depended on `save` being called.  The `if (deleteCMakeCache) {` block in `build` is currently unreachable.

NOTE: The deletion of `CMakeCache.txt` is a good idea, but we should do it completely as a new feature. CMake gives a reasonable error for cases it does need to be deleted, but it would be nice to automate (or semi-automate) that.